### DISCUSSION TITLE
Is rabbitmq-server vulnerable to CVE-2023-48795?

### DISCUSSION THREAD

**Discussion by nikita96** [rabbitmq/rabbitmq-server#10354] [Questions] [answered]
Erlang 25.6.2 is flagged for [CVE-2023-48795.](https://nvd.nist.gov/vuln/detail/CVE-2023-48795) SSH below 5.1.1 is affected. 
Is this CVE applicable if rabbitmq-server 3.11.10 is using erlang 25.6.2?
Please provide details.

**Comment by michaelklishin** [comment, marked as answer]
The question comes down to: does RabbitMQ use SSH for anything? No, it does not.
